Palindrome.java

```diff
@@ -0,0 +1,13 @@
+public boolean isPalindrome(String text) {
+    String clean = text.replaceAll("\\s+", "").toLowerCase();
+    int length = clean.length();
+    int forward = 0;
+    int backward = length - 1;
+    while (backward > forward) {
+        char forwardChar = clean.charAt(forward++);
```

Comment:
I find this part hard to understand. Could you please explain what happens here? (line 7 and 8)